new/LIT7221QeneBeelomu.xml

```diff
@@ -5,9 +5,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">ብዕሎሙ፡ውስተ፡ ሰማይ፡ ለነዳያነ፡ ምድር፡</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">Bǝʿǝlomu wǝsta samāy la-nadāyāna mǝdr (Qǝne possibly of wāzemā-type)</title>
-                <title xml:lang="en" corresp="#t1">At the feast for the poor of the country…</title>
+                <title xml:lang="gez" xml:id="t1">ብዕሎሙ፡ ውስተ፡ ሰማይ፡ ለነዳያነ፡ ምድር፡ ሐዋርያት፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Bǝʿlomu wǝsta samāy la-nadāyāna mǝdr ḥawāryāt... (Qǝne of śǝllāse-type)</title>
```

Comment:
I think the dominant variant is that we provide "..." only in the translation (not in the transliteration, because Ethiopic does not feature this). There are also some other minor differences.

Reply:
OK. I deleted the dots in the transliteration here and in some other records. I did not see any other minor differences from the preferable pattern, that you mentioned. Can you give me some hints? I will change this record and the others in sake of uniformity, then.